### ISSUE TITLE
CreateRubiasBaselineEval.GCL seed

### ISSUE THREAD

**Issue by csjalbert** [krshedd/GCL-R-Scripts#54]
We should add a seed to these baseline functions so the results are reproducible. `CreateRubiasBaselineEval.GCL` uses `foreach` and %dopar%  so for seeds to work in parallel we'll have to use the dorng package `https://cran.r-project.org/web/packages/doRNG/index.html` and replace the `dopar` with `dorng`

I can take care of this but wanted to make sure you guys were on board before adding this and have time to test it.

**Comment by awbarclay** [comment]
I’m on board with adding a seed to the function.  I’m pretty busy the next couple of weeks, so I might not be able to test it for a while.  Thanks for taking care of this!

Andy

**Comment by awbarclay** [comment]
I had time last Friday, so I updated the baseline eval functions with %dorng% and added seed arguments where needed. I updated and tested BaselineEvalSampleSizes, CreateRubiasBaselineEval, and run_rubias_baseline_eval.  In the tests I ran with the seeds set gave me the same result each time. These need more testing by others to make sure the seed argument and %dorng% foreach loops are working as expected.

Andy
